src/styles/utils/_variables.scss

```
@@ -0,0 +1,5 @@
+:root {
+  --primary-design-color: #00acdc;
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
```

```suggestion
 $blue: #00acdc;
```
you are using CSS variables. Use instead of them SASS variables